homeassistant/components/subaru/lock.py

```diff
@@ -0,0 +1,53 @@
+"""Support for Subaru door locks."""
```

Comment:
Since you should only have one platform, this file should be reviewed in the future PR in which it's added.

Reply:
Removed lock platform for this initial PR.